packages/manager/src/features/CloudPulse/shared/CloudPulseNodeTypeFilter.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { Autocomplete } from '@linode/ui';
+import * as React from 'react';
+
+import { useAllDatabasesQuery } from 'src/queries/databases/databases';
+
+import type { FilterValue } from '@linode/api-v4';
+
+export interface CloudPulseNodeTypes {
```

Comment:
```suggestion
export interface CloudPulseNodeType {
```

as you are defining this type of single option, so plural is not required